modules/engage-theodul-core/src/main/resources/ui/js/engage/models/meInfo.js

```diff
@@ -20,162 +20,78 @@
-                            else if ((key == prop_matomo_heartbeat) && value) {
-                              matomo_heartbeat = value;
-			    }
-                            else if ((key == prop_matomo_notification)) {
-                              if (value) {
-                              matomo_notification = value;
-                              } else matomo_notification = true; 
-                            }
-                            else if ((key == prop_matomo_track_events) && value) {
-                              matomo_track_events = value;
-                            }
-                        });
-                    }
-                    me.set("allowedtags", allowedTags);
-                    me.set("allowedformats", allowedFormats);
-                    me.set("shortcuts", shortcuts);
-                    me.set("mastervideotype", mastervideotype);
-                    me.set("logo_mediamodule", logo_mediamodule);
-                    me.set("logo_player", logo_player);
-                    me.set("link_mediamodule", link_mediamodule);
-                    me.set("show_embed_links", show_embed_link);
-                    me.set("hide_video_context_menu", hide_video_context_menu);
-                    me.set("shortcut-sequence", shortcut_sequence);
-                    me.set("layout", layout);
-                    me.set("focusedflavor", focusedflavor);
-                    me.set("matomo.server", matomo_server);
-                    me.set("matomo.site_id", matomo_site_id);
-                    me.set("matomo.heartbeat", matomo_heartbeat);
-                    me.set("matomo.notification", matomo_notification);
-                    me.set("matomo.track_events", matomo_track_events);
-                    ready = true;
-                }
-            });
-        },
-        ready: function() {
-            return ready;
-        },
-        getPositionControls: function() {
-            return positioncontrols;
-        }
-    });
+          $.ajax({
+            url: configURL,
+            dataType: "text",
+            success: function (data) {
+              var rawfile = data;
+            }
+          }).then(function (rawfile) {
+            config = jsyaml.load(rawfile);
+            me.set("config", config);
+            me.set("allowedtags", config.allowedTags);
+            me.set("allowedformats", config.allowedFormats);
+            me.set("shortcuts", config.shortcuts);
+            me.set("mastervideotype", config.mastervideotype);
+            me.set("logo_mediamodule", config.logo_mediamodule);
+            me.set("logo_player", config.logo);
+            me.set("show_embed_links", config.show_embed_link);
+            me.set("hide_video_context_menu", config.hide_video_context_menu);
+            me.set("layout", config.layout);
+            me.set("focusedflavor", config.focusedflavor);
+
+            if (config.matomo) {
+              me.set("matomo.server", config.matomo_server);
```

Comment:
This should still be `config.matomo.server ...` (note the dot after `matomo`).

Reply:
I see, I suppose the others also needs to be replaced the "_" with a point "." Is that correct?

```javascript
if (config.matomo) {
              me.set("matomo.server", config.matomo_server);
              me.set("matomo.site_id", config.matomo_site_id);
              me.set("matomo.heartbeat", config.matomo_heartbeat);
              me.set("matomo.notification", config.matomo_notification);
              me.set("matomo.track_events", config.matomo_track_events);
            }
```

Reply:
@mtneug

Reply:
Sorry, I didn't notice your comment.

`me.set("matomo.server", config.matomo.server); ...` should be correct.

Reply:
Ok, Thanks :)

Reply:
I'm sorry with `...` I also meant to apply the change to all Matomo settings:
```js
me.set("matomo.server", config.matomo.server);
me.set("matomo.site_id", config.matomo.site_id);
me.set("matomo.heartbeat", config.matomo.heartbeat);
me.set("matomo.notification", config.matomo.notification);
me.set("matomo.track_events", config.matomo.track_events);
```

Reply:
Ahh ok, I was confused. I will solve in a minute ;)